lib/helpers/remote_control_helper.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require 'net/ssh'
+require 'net/sftp'
+require 'stringio'
+require 'logger'
+
+# @return [Logger]
+def logger
+  @logger = Logger.new($stdout)
+end
+
+# @param [Object] host
+# @param [Object] user
+# @param [Proc] block
+#
+# @return [Array, Net::SSH::Authentication]
+def ssh(host, user, &block)
```

Comment:
This is a global method, it's not a good idea to have a global method in project, maybe it should be inside some class?